pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -1,4 +1,5 @@
 %link{:href=>"/shared/css/warning-banner.css", :rel=>"stylesheet"}
+%script{src: minifiable_asset_path("js/pegasusCommon.js")}
```

Comment:
Honestly the separation between themes and layouts does seem muddy and I'm not sure how it got to that point (a number of people had different philosophies about factoring in pegasus page construction).  But basically we start with the page body itself, wrap that in the layout, and then wrap that in the theme.  (The `= body` in the HAML for layout and theme is how they wrap their content.)  But both theme and layout are optional, so it comes down to how the pages are constructed.  https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/router.rb#L226-L239

From eyeballing the layouts and themes in use for code.org, it does appear that `theme_common_header_after` could be included in all the right places.